src/index.js

```diff
@@ -1,13 +1,15 @@
-import directives from './directives';
+import directives from "./directives";
 
 const VAnimateCss = {
   install (Vue, options = {}) {
     const animateCSSPath = options.animateCSSPath;
-    const link = document.createElement('link');
-    link.rel = 'stylesheet';
-    link.href = animateCSSPath || 'https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css' ;
-    document.getElementsByTagName('head')[0].appendChild(link);
-    
+    const link = document.createElement("link");
+    link.rel = "stylesheet";
+    link.href =
+      animateCSSPath ||
+      "https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css";
```

Comment:
Please put in a constant variable above `const VAnimateCss` for better readability. Like so:

```js
const ANIMATE_CSS_CDN_LINK = 'https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css';

const VAnimateCass = ....
```